content/post/Android_Root101.en.md

```diff
@@ -39,26 +42,28 @@ Place this file inside the folder created after extraction.
 
 * Now open up the command prompt on Windows with **_adminitrator priviliges_**.
 
-    _(To do this press the Windows key on your keyboard and search 'cmd' right click when command prompt comes up and chose 'Run as Administrator')_
+    _(To do that, press the `Windows key` on your keyboard and search 'cmd', right click when command prompt comes up and chose 'Run as Administrator')_
 
 * Now go the directory where the files you downloaded have been stored.
     if you don't know how to use Command Prompt
 
 > `cd <dirname>` is use to go up a directory
 
-> `cd..` is used to go down one directory
+> `cd ..` is used to go down one directory
 
+Enable Developer Options on your phone and enable USB-Debugging. To know more [Google Search](https://www.google.com)
```

Comment:
Woah
```
(https://www.google.com) ??
```
Also, better to link an article than a google search
If people have to go out of the article to search for related stuff, why would they read your article? there are a lot more articles available which describe everything.
**Never** make your audience leave your page